model/components/icinga.yml

```diff
@@ -0,0 +1,31 @@
+---
+components:
+
+  # Icinga 2
+  - id: icinga2
+    name: Icinga 2
+    description: Icinga monitors your network, servers and applications in a secure and reliable way.
+    link: https://www.icinga.com
+    github:
+      - icinga/icinga2
+    twitter: icinga
+    logo: icinga
+    license:
+      - GPLv2
+    categories:
+      - agent
+      - collector
+      - visualization
+      - dashboarding
+      - alerting
+    capabilities:
+      aspects:
+        - metrics
```

Comment:
Can you add what sub-category of metrics are you fulfilling? Currently we have `metrics [app]`, `metrics [process]` and  `metrics [host]`. 

This way tool would still appear in the generic metric filter, but also when `metrics [host]` is selected (if host is added). I believe this is way better than just metrics. If these 3 metrics sub-categories are not enough for you, you can also suggest new ones adn we can disucss it.

Other than that PR is fine, we can merge it.. Thanks for contrib!

Reply:
Thanks for the feedback, the metrics aspects available are just fine, I will provide an update.

Regarding the general `categories`: Are there any plans to add a category for availability monitoring?